docs/organizations/managing-repositories.md

```diff
@@ -1,10 +1,12 @@
 # Managing repositories
 
-To see a list of the repositories that Codacy is monitoring, open the page **Repositories** under your organization.
+Users with the [necessary permissions on your Git provider](roles-and-permissions-for-organizations.md) can **add** repositories to Codacy to start analyzing them. The remaining organization members can then **follow** added repositories of their interest on Codacy.
```

Comment:
Minor nitpick here, the ability to follow is also tied to necessary permission on the Git provider. It's not completely clear from the current phrasing. This is not blocking in any way.

Reply:
You're right, @jorgebraz. Thanks for the input! 👍
I tweaked the sentence to clarify that the remaining users also need access to the repositories they want to follow.